File: Chapter 1/(d)/d.java
import java.util.Scanner;

class d
{
    static public void main(String args[])
    {
        float tempF;
        System.out.println("Enter the tenperature in Fahrenheit ");
        Scanner input=new Scanner(System.in);
        tempF=input.nextFloat();
        System.out.println("Temperature in Centigrades is "+(tempF-32)*5/9);
    }
}

File: Chapter 1/(d)/d.c
#include <stdio.h>

int main(int argc, char *args[])
{
	float tempF;
	printf("Enter the temperature in Farenheit : ");
	scanf("%f", &tempF);
	printf("Temperature in Centigrades is : %0.3f C", (tempF - 32) * 5 / 9);

	return 0;
}